xsd/ServerConfig.xsd

```diff
@@ -71,8 +72,24 @@
             <xs:element name="SecuritySetting" maxOccurs="unbounded">
                 <xs:complexType>
                     <xs:sequence>
-                        <xs:element name="SecurityPolicy" type="xs:string"></xs:element>
-                        <xs:element name="MessageSecurityMode" type="xs:string"></xs:element>
+                        <xs:element name="SecurityPolicy">
+                            <xs:simpleType>
+                                <xs:restriction base="xs:string">
+                                    <xs:enumeration
+                                        value="http://opcfoundation.org/UA/SecurityPolicy#None"></xs:enumeration>
+                                    <xs:enumeration
+                                        value="http://opcfoundation.org/UA/SecurityPolicy#Basic128Rsa15"></xs:enumeration>
+                                </xs:restriction>
```

Comment:
worth adding these for completeness ?

![image](https://user-images.githubusercontent.com/319874/76338627-3db6f280-62f9-11ea-9a75-b85557fa0d1d.png)

Reply:
Thanks @ben-farnham , it was taken into account.